[2023-02-27T23:45:20.459+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-02-27T23:45:17.473925+00:00 [queued]>
[2023-02-27T23:45:20.466+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-02-27T23:45:17.473925+00:00 [queued]>
[2023-02-27T23:45:20.466+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-27T23:45:20.466+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-02-27T23:45:20.466+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-27T23:45:20.477+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-02-27 23:45:17.473925+00:00
[2023-02-27T23:45:20.481+0000] {standard_task_runner.py:55} INFO - Started process 2963 to run task
[2023-02-27T23:45:20.483+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'manual__2023-02-27T23:45:17.473925+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpuuzzdllk']
[2023-02-27T23:45:20.483+0000] {standard_task_runner.py:83} INFO - Job 43: Subtask insert_into_table
[2023-02-27T23:45:20.529+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-02-27T23:45:17.473925+00:00 [running]> on host 971ca163b440
[2023-02-27T23:45:20.588+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=temi109
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2023-02-27T23:45:17.473925+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-27T23:45:17.473925+00:00
[2023-02-27T23:45:20.589+0000] {sql.py:254} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-02-27', 'dag_with_postgres_operator_v03')
        
[2023-02-27T23:45:20.595+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-02-27T23:45:20.770+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-02-27T23:45:20.779+0000] {sql.py:375} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-02-27', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-02-27T23:45:20.782+0000] {sql.py:384} INFO - Rows affected: 1
[2023-02-27T23:45:20.792+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20230227T234517, start_date=20230227T234520, end_date=20230227T234520
[2023-02-27T23:45:20.815+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-27T23:45:20.830+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
